sections/main-cart-footer.liquid

```diff
@@ -64,14 +64,30 @@
                 </div>
 
                 <small class="tax-note caption-large rte">
-                  {%- if cart.taxes_included and shop.shipping_policy.body != blank -%}
-                    {{ 'sections.cart.taxes_included_and_shipping_policy_html' | t: link: shop.shipping_policy.url }}
-                  {%- elsif cart.taxes_included -%}
-                    {{ 'sections.cart.taxes_included_but_shipping_at_checkout' | t }}
-                  {%- elsif shop.shipping_policy.body != blank -%}
-                    {{ 'sections.cart.taxes_and_shipping_policy_at_checkout_html' | t: link: shop.shipping_policy.url }}
-                  {%- else -%}
-                    {{ 'sections.cart.taxes_and_shipping_at_checkout' | t }}
+                  {%- if cart.duties_included == true and cart.taxes_included == true -%}
```

Comment:
```suggestion
                  {%- if cart.duties_included and cart.taxes_included -%}
```
This is logically the same right?

Reply:
Yup you are right, I think we can also do `!` instead of `== false` as well. I will clean this logic up